tests/suites/test_suite_psa_crypto_util.data

```diff
@@ -1,65 +1,65 @@
 ECDSA Raw -> DER, 256bit, Success
-depends_on:PSA_WANT_ECC_SECP_R1_256
+depends_on:PSA_WANT_ECC_SECP_K1_256
```

Comment:
Do these test really depend on a curve being enabled, though? It's just format conversion, I don't see where curve knowledge is needed, so perhaps we could just remove those dependencies? Or am I missing something?

Reply:
In functions `mbedtls_ecdsa_raw_to_der` and `mbedtls_ecdsa_der_to_raw` we need to know:
- `PSA_VENDOR_ECC_MAX_CURVE_BITS` and
- `PSA_VENDOR_ECDSA_SIGNATURE_MAX_SIZE`
respectively. So yes, curves' types definitions are basically used only to properly size buffers.

Reply:
Ah, right, that's what I missed :) Thanks for the explanation!